public/mix-manifest.json

```
@@ -0,0 +1,4 @@
+{
+    "/js/app.js": "/js/app.js",
+    "/css/app.css": "/css/app.css"
+}
```

two space indentation for json

```suggestion
{
  "/js/app.js": "/js/app.js",
  "/css/app.css": "/css/app.css"
}
```